dcc-portal-ui/package.json

```diff
@@ -23,7 +24,7 @@
     "grunt": "0.4.5",
     "grunt-bower-install-simple": "1.2.3",
     "grunt-concurrent": "0.3.0",
-    "grunt-connect-proxy": "0.2.0",
+    "grunt-connect-proxy": "git+https://github.com/drewzboto/grunt-connect-proxy.git#f798bbd31b76b039a392b8f1bca55b310a7ac5c9",
```

Comment:
Not quite, this pins it to a specific commit from May
https://github.com/drewzboto/grunt-connect-proxy/commit/f798bbd31b76b039a392b8f1bca55b310a7ac5c9

The module author hasn't published updates to the npm module for over a year, it's rather out of date/sync with the repo

Maybe lost motivation to continue maintenance duties